src/frameworks/drupal7/simplesaml.md

```diff
@@ -96,6 +101,22 @@ if (isset($_ENV['PLATFORM_PROJECT_ENTROPY'])) {
 }
 ```
 
+## Generate SSL certs (optional)
+
+Depending on your Identity Provider (IdP), you may need to generate an SSL/TLS certificate to connect to the SP.  If so, you should generate the cert locally following the instructions in the [SimpleSAMLphp documentation](https://simplesamlphp.org/docs/stable/simplesamlphp-sp).  Whatever your resulting idP file is should be placed in the simplesamlphp/metadata directory.  The certificate should be placed in the `simplesamlphp/cert` directory.  (Create it if needed.)
+
+Then add the following line to your `simplesamlphp/config/config.php` file to tell the library where to find the cert:
+
+```php
+$config['certdir'] = dirname(__DIR__) . '/cert';
```

Comment:
`dirname(__DIR__)`?

Reply:
Shortest way to get the parent directory.  dirname() takes a path and trims off the last part, effectively giving you `__DIR__ . '/..'`.  (I didn't realize it either, but the customer I was working with on these docs found it worked, and it's nice and short!)